example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -46,53 +46,59 @@ describe('The Ingest Granule failure workflow', () => {
-    const provider = { id: `s3_provider${testSuffix}` };
-
-    process.env.GranulesTable = `${config.stackName}-GranulesTable`;
-    granuleModel = new Granule();
-
-    process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
-    executionModel = new Execution();
-
-    // populate collections, providers and test data
-    await Promise.all([
-      uploadTestDataToBucket(config.bucket, s3data, testDataFolder),
-      addCollections(config.stackName, config.bucket, collectionsDir, testSuffix, testId),
-      addProviders(config.stackName, config.bucket, providersDir, config.bucket, testSuffix),
-    ]);
-
-    const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
-    // update test data filepaths
-    inputPayload = await setupTestGranuleForIngest(config.bucket, inputPayloadJson, granuleRegex, testSuffix, testDataFolder);
-
-    // add a non-existent file to input payload to cause lambda error
-    const nonexistentFile = {
-      path: 'non-existent-path',
-      name: 'non-existent-file',
-      bucket: 'non-existent-bucket',
-      key: 'non-existent-key',
-    };
-    inputPayload.granules[0].files.push(nonexistentFile);
-
-    // delete the granule record from DynamoDB if exists
-    await granuleModel.delete({ granuleId: inputPayload.granules[0].granuleId });
-
-    workflowExecution = await buildAndExecuteWorkflow(
-      config.stackName,
-      config.bucket,
-      workflowName,
-      collection,
-      provider,
-      inputPayload
-    );
+    try {
+      config = await loadConfig();
+      const testId = createTimestampedTestId(config.stackName, 'IngestGranuleFailure');
+      testSuffix = createTestSuffix(testId);
+      testDataFolder = createTestDataPath(testId);
+
+      const collection = { name: `MOD09GQ${testSuffix}`, version: '006' };
+      const provider = { id: `s3_provider${testSuffix}` };
+
+      process.env.GranulesTable = `${config.stackName}-GranulesTable`;
+      granuleModel = new Granule();
+
+      process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
+      executionModel = new Execution();
+
+      // populate collections, providers and test data
+      await Promise.all([
+        uploadTestDataToBucket(config.bucket, s3data, testDataFolder),
+        addCollections(config.stackName, config.bucket, collectionsDir, testSuffix, testId),
+        addProviders(config.stackName, config.bucket, providersDir, config.bucket, testSuffix),
+      ]);
+
+      const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
+      // update test data filepaths
+      inputPayload = await setupTestGranuleForIngest(config.bucket, inputPayloadJson, granuleRegex, testSuffix, testDataFolder);
+
+      // add a non-existent file to input payload to cause lambda error
+      const nonexistentFile = {
+        path: 'non-existent-path',
+        name: 'non-existent-file',
+        // bucket: 'non-existent-bucket',
+        // key: 'non-existent-key',
+      };
```

Comment:
I think this is the problem. Logging the response from `invokeApi` and `getExecution` show an error that points pretty explicitly to this file (below). 

```
in invokeApi. parsedPayload:::::::::
--
23-Apr-2021 17:07:39 | {
23-Apr-2021 17:07:39 | errorType: 'Runtime.UnhandledPromiseRejection',
23-Apr-2021 17:07:39 | errorMessage: 'NoSuchBucket: The specified bucket does not exist; Function params: [\n' +
23-Apr-2021 17:07:39 | '  "non-existent-bucket",\n' +
23-Apr-2021 17:07:39 | '  "non-existent-key"\n' +
23-Apr-2021 17:07:39 | ']',
23-Apr-2021 17:07:39 | trace: [
23-Apr-2021 17:07:39 | 'Runtime.UnhandledPromiseRejection: NoSuchBucket: The specified bucket does not exist; Function params: [',
23-Apr-2021 17:07:39 | '  "non-existent-bucket",',
23-Apr-2021 17:07:39 | '  "non-existent-key"',
23-Apr-2021 17:07:39 | ']',
23-Apr-2021 17:07:39 | '    at process.<anonymous> (/var/runtime/index.js:35:15)',
23-Apr-2021 17:07:39 | '    at process.emit (events.js:314:20)',
23-Apr-2021 17:07:39 | '    at process.emit (/var/task/node_modules/source-map-support/source-map-support.js:495:1)',
23-Apr-2021 17:07:39 | '    at processPromiseRejections (internal/process/promises.js:209:33)',
23-Apr-2021 17:07:39 | '    at processTicksAndRejections (internal/process/task_queues.js:98:32)'
23-Apr-2021 17:07:39 | ]
23-Apr-2021 17:07:39 | }
23-Apr-2021 17:07:39 | in getExecution. response:::::::
23-Apr-2021 17:07:39 | {
23-Apr-2021 17:07:39 | errorType: 'Runtime.UnhandledPromiseRejection',
23-Apr-2021 17:07:39 | errorMessage: 'NoSuchBucket: The specified bucket does not exist; Function params: [\n' +
23-Apr-2021 17:07:39 | '  "non-existent-bucket",\n' +
23-Apr-2021 17:07:39 | '  "non-existent-key"\n' +
23-Apr-2021 17:07:39 | ']',
23-Apr-2021 17:07:39 | trace: [
23-Apr-2021 17:07:39 | 'Runtime.UnhandledPromiseRejection: NoSuchBucket: The specified bucket does not exist; Function params: [',
23-Apr-2021 17:07:39 | '  "non-existent-bucket",',
23-Apr-2021 17:07:39 | '  "non-existent-key"',
23-Apr-2021 17:07:39 | ']',
23-Apr-2021 17:07:39 | '    at process.<anonymous> (/var/runtime/index.js:35:15)',
23-Apr-2021 17:07:39 | '    at process.emit (events.js:314:20)',
23-Apr-2021 17:07:39 | '    at process.emit (/var/task/node_modules/source-map-support/source-map-support.js:495:1)',
23-Apr-2021 17:07:39 | '    at processPromiseRejections (internal/process/promises.js:209:33)',
23-Apr-2021 17:07:39 | '    at processTicksAndRejections (internal/process/task_queues.js:98:32)'
23-Apr-2021 17:07:39 | ]
23-Apr-2021 17:07:39 | }


```